core-server/src/main/java/org/orbisgis/orbisserver/coreserver/web/MainController.java

```diff
@@ -136,7 +136,19 @@ public Result login() throws IOException {
     public Result processList(@Parameter("token") String token) throws IOException {
         for(Session session : sessionList) {
             if (session.getToken().toString().equals(token)) {
-                return ok(render(processList, "processList", session.getOperationList()));
+                List<Operation> opList = session.getOperationList();
+                List<Operation> importExportList = new ArrayList<Operation>();
+
+                for(Operation op : opList){
+                    for(String keyword :  op.getKeyWord()){
+                        int index = 0;
+                        if(keyword.equals("Export") || keyword.equals("Import")){
+                            importExportList.add(op);
```

Comment:
Why not having two list, one for the import, and an other for the export. I only one list is kept, maybe the name should be changed to avoid confusion.